src/components/content/order/serviceConfiguration/ServiceConfigurationDetails.tsx

```diff
@@ -0,0 +1,199 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { CheckCircleOutlined, CloseCircleOutlined, EditOutlined } from '@ant-design/icons';
+import { Button, Empty, Form, Input, Popconfirm, Row } from 'antd';
+import React, { useEffect, useState } from 'react';
+import '../../../../styles/app.module.css';
+import serviceOrderStyles from '../../../../styles/service-order.module.css';
+import {
+    type ChangeServiceConfigurationData,
+    DeployedServiceDetails,
+    ServiceConfigurationUpdate,
+    serviceHostingType,
+    taskStatus,
+    UserOrderableServiceVo,
+    VendorHostedDeployedServiceDetails,
+} from '../../../../xpanse-api/generated';
+import { useLatestServiceOrderStatusQuery } from '../../common/queries/useLatestServiceOrderStatusQuery.ts';
+import { ServiceConfigParameterItem } from './ServiceConfigParameterItem.tsx';
+import ServiceConfigStatusAlert from './ServiceConfigStatusAlert.tsx';
+import useGetCurrentConfigurationOfServiceQuery from './useGetConfigurationOfServiceQuery.ts';
+import { useUpdateServiceConfigurationRequest } from './useUpdateServiceConfigurationRequest.ts';
+
+export const ServiceConfigurationDetails = ({
+    userOrderableServiceVo,
+    deployedService,
+}: {
+    userOrderableServiceVo: UserOrderableServiceVo | undefined;
+    deployedService: DeployedServiceDetails | VendorHostedDeployedServiceDetails;
+}): React.JSX.Element => {
+    const [form] = Form.useForm();
+    const [isUpdateConfig, setIsUpdateConfig] = useState<boolean>(false);
+    const [isShowUpdateConfigResult, setIsShowUpdateConfigResult] = useState<boolean>(false);
+
+    const getCurrentConfigurationOfServiceQuery = useGetCurrentConfigurationOfServiceQuery(deployedService.serviceId);
+
+    const updateConfigRequest = useUpdateServiceConfigurationRequest();
+
+    const getUpdateConfigStatusPollingQuery = useLatestServiceOrderStatusQuery(
+        updateConfigRequest.data?.orderId ?? '',
+        updateConfigRequest.isSuccess,
+        [taskStatus.SUCCESSFUL, taskStatus.FAILED]
+    );
+
+    const updateConfig = (values: ServiceConfigurationUpdate) => {
+        setIsShowUpdateConfigResult(true);
+        const data: ChangeServiceConfigurationData = {
+            requestBody: { configuration: values },
+            serviceId: deployedService.serviceId,
+        };
+        updateConfigRequest.mutate(data, {
+            onSuccess: () => {
+                void getCurrentConfigurationOfServiceQuery.refetch();
```

Comment:
do we really need a refetch here? becaus the configuration is not updating as soon as mutate is successful.